src/arguments/membername.ts

```diff
@@ -10,7 +10,7 @@ export default class extends Argument {
 		const resMember = await this.resolveMember(message, arg);
 		if (resMember) return resMember;
 
-		const result = await new FuzzySearch(message.guild!.memberUsernames, entry => entry, filter).run(message, arg, possible.min || undefined);
+		const result = await new FuzzySearch(message.guild!.nicknames.mapUsernames(), entry => entry, filter).run(message, arg, possible.min || undefined);
```

Comment:
I thought DJS fixed the need for the non null assertion for message.guild? Same for username if so.

Reply:
https://github.com/kyranet/Skyra/blob/4cb98c2aa110d0ef4ae2aa062920acf27705635d/src/commands/Tools/poll.ts#L147

Returns `string | null`.

Reply:
Messages sent in a DM channel have `guild === null`.